packages/base/core/src/less/variables/components/sidebar.less

```diff
@@ -1,4 +1,5 @@
 // Sidebar
+@oui-sidenar-padding: rem-calc(12);
```

Comment:
@AxelPeter I'm suspecting a typo, isn't it?

`sidenar` => `sidebar`

Reply:
Yes, but working since it uses the variables with the typo. Fixing that in another commit ;)